templates/01_block-scoping.test.js

```diff
@@ -92,5 +95,21 @@ test('means that we can start using block statements', () => {
 
 //////// EXTRA CREDIT ////////
 
+test.skip('means that we can declare constant with the same name in block statement', () => {
+  // WORKSHOP_START
+  // Declare 'd' using 'const', setting the value to 5
+  // In a block statement declare 'd' using 'const', setting the value to 10
+  expect(d).toBe(/* ENTER YOUR GUESS HERE */)
+  // WORKSHOP_END
+  // FINAL_START
+  const d = 5
+  // BLOCK STATEMENT
+  {
+    const d = 10
+  }
+  expect(d).toBe(5)
+  // FINAL_END
```

Comment:
Same idea for this test as well. I think it'd be confusing for them to read what they should be coding. Unless they had an actual use-case... What if we did something like:

```javascript
// Declare a `d` here with `const`
{
  // Declare a `d` here with `const`
  expect(d).toBe(10)
}
expect(d).toBe(5)
```

Or something like that. This way they're still writing out the `const` themselves, but we can assert that they're doing things correctly by putting it in a block. Thoughts?